doc/build-unix.md

```diff
@@ -69,7 +69,7 @@ Build requirements:
 
 Options when installing required Boost library files:
 
-1. On at least Ubuntu 14.04+ and Debian 7+ there are generic names for the
+1. On at least Ubuntu 14.04+ ( > kernel 3.13 )  and Debian 7+( > kernel v3.2.41)  there are generic names for the
```

Comment:
A wide range of Linux distributions, even different OSes (Android) share the same kernel. Generally libc shields the details of the kernel and its interface completely from user space software and libraries such as Bitcoin Core and Boost.

What matters for this specific advice is what *Linux distribution* one uses. Specifically the package system. The version of boost used is completely independent of the kernel, and so is how you install boost.

Mentioning a kernel version here is just confusing and just sends people into the woods. Please remove.